implementations/code_df/commit.py

```diff
@@ -1,14 +1,16 @@
 import pandas as pd
-from .metric import Metric
-from . import utils
+
+import conditions
+from metric import Metric
+import utils
```

Comment:
There is a problem with importing modules this way. Since these modules are imported into the reference notebooks, which are in a different directory, using a relative import works. The new changes give a `ModuleNotFound` exception.
Python does not check for a modules location in the parent directory. As I am not a hundred percent sure of the fix, I'm open for inputs @valeriocos @aswanipranjal. :)

My current solution is to add `..` to sys.path and then change the intra package imports inside `code_df` (like in  commits.py) to relative imports. 

**notebook:**
```python
import sys
sys.path.append('..')
```
**commits.py:**
```python
from . import conditions
from . import utils
from .commit import Commit
```

Reply:
@valeriocos , could you have a look at this? Thanks

Reply:
Which directory are you running the script from, @Polaris000?

Reply:
notebooks_df

Reply:
for the notebooks, I think
```
import sys
sys.path.append('..')
```

would be the way to go

Reply:
For that to work, commit.py should also be changed (as I mentioned above) so that it uses relative imports. Thanks @aswanipranjal!

Reply:
correct, @Polaris000.

Reply:
Thanks a lot for all this discussion. If you don't mind, let's keep it as it is now, and let's figure out in a following pr how to deal with this in the context of Notebooks. When I proposed to have separate directories, I didn't notice this problem. Maybe we can revert to having just one directory, or find some other solution... Suggestions are welcome ;-)